index.js

```diff
@@ -568,19 +568,8 @@ app.post('/api/sirenEscalated', Validator.body(['coreid']).exists(), async (requ
 
           const session = await db.getUnrespondedSessionWithLocationId(location.locationid, client)
           if (session) {
-            const alertInfo = {
-              sessionId: session.id,
-              toPhoneNumber: location.responderPhoneNumber,
-              fromPhoneNumber: location.twilioNumber,
-              message: `This is an unresponded siren alert. Please check on the bathroom at ${location.displayName}. Please respond with 'ok' once you have checked on it.`,
-              reminderTimeoutMillis: location.reminderTimer,
-              fallbackTimeoutMillis: location.fallbackTimer,
-              reminderMessage: `This is a reminder to check on the bathroom`,
-              fallbackMessage: `An alert to check on the bathroom at ${location.displayName} was not responded to. Please check on it`,
-              fallbackToPhoneNumbers: location.fallbackNumbers,
-              fallbackFromPhoneNumber: location.client.fromPhoneNumber,
-            }
-            braveAlerter.startAlertSession(alertInfo)
+            alertMessage= 'There is an unresponded siren. Please check on the bathroom at ${location.displayName}.'
+            braveAlerter.sendSingleAlert(location.responderPhoneNumber,location.twilioNumber,alertMessage)
```

Comment:
Can you please change this to send it to the fallback phone numbers? It'll be an array of numbers, so you'll probably need to loop over them. The sender number should be `location.client.fromPhoneNumber` instead of `location.twilioNumber`.

There also needs to be a space after each comma in `(location.responderPhoneNumber,location.twilioNumber,alertMessage)` and a space before the equal sign in ` alertMessage= `.

You also need to use backquotes instead of single quotes in `'There is an unresponded siren. Please check on the bathroom at ${location.displayName}.'` in order for the `${}` part to work.